dependencycheck.properties

```diff
@@ -1,13 +1,19 @@
 category=Integration
 description=Integrates Dependency-Check reports into SonarQube
 homepageUrl=https://github.com/dependency-check/dependency-check-sonar-plugin
-archivedVersions=2.0.7,2.0.8,3.0.0,3.0.1
-publicVersions=3.1.0
+archivedVersions=2.0.7,2.0.8,3.0.0,3.0.1,3.1.0
+publicVersions=4.0.0
 defaults.mavenGroupId=org.sonarsource.owasp
 defaults.mavenArtifactId=sonar-dependency-check-plugin
 
+4.0.0.description=Support SonarQube 10
+4.0.0.sqVersions=[9.9.1, LATEST]
```

Comment:
Do you really mean to exclude users on 9.9(.0)? Unless that's a deliberate exclusion, this should start at 9.9

Reply:
You're right that the latest point version is considered the LTS. And unfortunately, that doesn't mean that everyone upgrades to it immediately. By starting this at 9.9, people who haven't upgraded the server yet will still be prompted to upgrade their version of your plugin.